languages/javascript/exercises/concept/arrays/arrays.js

```diff
@@ -1,23 +1,79 @@
+/**
+ * @param {number[]} array
+ * @param {number} index
+ * 
+ * @returns {number[]}
```

Comment:
```suggestion
 * @returns {number} the item at position `index`  in the given `array`
```

Reply:
Alternatively:

```suggestion
 * @returns {number}
```

And add a short description (1 line) to each function, at the top.